tests/e2e/test_args.sh

```diff
@@ -0,0 +1,12 @@
+cd $GOPATH/src/k8s.io/kubernetes
```

Comment:
Please move the file to hack/test-k8s-e2e.sh

Also please include the headers (you can copy from other files) and make the file as executable (chmod +x <file>).

Reply:
okk Sir

Reply:
I added those lines
But  it is already executable.